bundle/manifests/external-dns-operator_clusterserviceversion.yaml

```diff
@@ -122,8 +122,8 @@ spec:
   maturity: alpha
   provider:
     name: Red Hat, Inc.
-  version: 0.0.1
-  minKubeVersion: 1.22
+  version: "0.0.1"
+  minKubeVersion: "1.22.0"
```

Comment:
does this need to comply with semver ?

Reply:
Good question, I don't know for sure, I just looked up some examples for other operator bundles.